tests/integration/gateway_clients/test_streaming.py

```diff
@@ -177,17 +177,6 @@ def create_runtime(graph_dict: Dict, protocol: str, port_in: int, prefetch: int
     'protocol, inputs',
     [
         ('grpc', slow_async_gen),
-        pytest.param(
```

Comment:
why did u remove all this? It should be added back

Reply:
true, probably I debugged sth and forgot to readd, will do